test/EstateRegistry.js

```diff
@@ -493,13 +493,43 @@ contract('EstateRegistry', accounts => {
     })
   })
 
-  describe('land update', function() {
-    it('should not allow old owner to update land after creating an Estate', async function() {
+  describe('LAND update', function() {
+    it('should allow owner of an Estate to update LAND data', async function() {
+      await land.assignMultipleParcels([0], [1], user, sentByCreator)
+      const estateId = await createEstate([0], [1], user, sentByUser)
+      await estate.updateLandData(estateId, 0, 1, 'newValue', sentByUser)
+      const data = await land.landData(0, 1, sentByUser)
+      data.should.be.equal('newValue')
+    })
+    it('should allow operator of an Estate to update LAND data', async function() {
+      await land.assignMultipleParcels([0], [1], user, sentByCreator)
+      await createEstate([0], [1], user, sentByUser)
+      await estate.setApprovalForAll(anotherUser, true, sentByUser)
+      await estate.updateLandData(1, 0, 1, 'newValue', sentByAnotherUser)
+      const data = await land.landData(0, 1, sentByUser)
+      data.should.be.equal('newValue')
+    })
+    it('should allow update operator of an Estate to update LAND data', async function() {
+      await land.assignMultipleParcels([0], [1], user, sentByCreator)
+      await createEstate([0], [1], user, sentByUser)
+      await estate.setUpdateOperator(1, anotherUser, sentByUser)
+      await estate.updateLandData(1, 0, 1, 'newValue', sentByAnotherUser)
+      const data = await land.landData(0, 1, sentByUser)
+      data.should.be.equal('newValue')
+    })
+    it('should not allow neither operator, nor owner nor updateOperator of an Estate to update LAND data', async function() {
+      await land.assignMultipleParcels([0], [1], user, sentByCreator)
+      await createEstate([0], [1], user, sentByUser)
+      await assertRevert(
+        estate.updateLandData(1, 0, 1, 'newValue', sentByAnotherUser)
```

Comment:
Following my question above, a test might be similar to this but look something like this:

```
it('should not allow neither operator, nor owner nor updateOperator of an Estate to update LAND data', async function() {
  // Assign LAND 0,1 to user
  await land.assignMultipleParcels([0], [1], user, sentByCreator)
  // Transfer 0,1 to first estate of estateId === 1
  const estateId = await createEstate([0], [1], user, sentByUser)

  // Assign LAND 1,1 to anotherUser
  await land.assignMultipleParcels([1], [1], anotherUser, sentByCreator)
  // Transfer 1,1 to first estate of estateId === 2
  await createEstate([1], [1], user, sentByUser)

  // Try to update 0,1 with the first estateId
  await assertRevert(
    estate.updateLandData(estateId, 0, 1, 'newValue', sentByAnotherUser)
  )
})
```